reference/3.0/Microsoft.PowerShell.Core/Enable-PSRemoting.md

```diff
@@ -36,14 +37,14 @@ For more information, see the description of the **SkipNetworkProfileCheck** par
 The **Enable-PSRemoting** cmdlet performs the following operations:
 
 - Runs the [Set-WSManQuickConfig](../Microsoft.WsMan.Management/Set-WSManQuickConfig.md) cmdlet, which performs the following tasks:
- - Starts the WinRM service.
- - Sets the startup type on the WinRM service to Automatic.
- - Creates a listener to accept requests on any IP address.
- - Enables a firewall exception for WS-Management communications.
- - Registers the Microsoft.PowerShell and Microsoft.PowerShell.Workflow session configurations, if it they are not already registered.
- - Registers the Microsoft.PowerShell32 session configuration on 64-bit computers, if it is not already registered.
- - Enables all session configurations.
- - Changes the security descriptor of all session configurations to allow remote access.
+    - Starts the WinRM service.
```

Comment:
Please read the Style Guide I linked to yesterday. The indentation for sub-items should line up with the first character of the parent item. This is the new standard in the CommonMark spec for list items.

Reply:
https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/contributing/4-MARKDOWN-SPECIFICS.md#lists

Reply:
I'll look at adding an EditorConfig for the indent size separately. This list was only indented 1 space, and VS Code guessed the size as 4

Reply:
Right, the original was not indented correctly. Your fix would technically work in our publishing system but we are trying to match the standards to avoid future problems. Thanks for your help.

Reply:
Rebased this and sent a separate PR for EditorConfig